tests/test_tls/test_tls.c

```diff
@@ -27,26 +27,6 @@
  * the context of Solo5 means at all.
  */
 
-/*
- * FreeBSD clang toolchain post 8.0.1/12.1-RELEASE fails to build this test
- * with:
- *
- * ld: error: test_tls.o has an STT_TLS symbol but doesn't have an SHF_TLS section
- *
- * OpenBSD clang toolchains have unspecified non-support for TLS.
```

Comment:
The CI failure may be related to that comment. Is there a way to check for OpenBSD in the C source and push back the #ifdef ?

Reply:
I'd guess `#if defined(__OpenBSD__)`?

Reply:
Indeed, thanks!